src/main/java/cs505/grad1/sensoragg/mock/MockFactory.java

```diff
@@ -0,0 +1,36 @@
+package cs505.grad1.sensoragg.mock;
+
+import java.util.Map;
+import java.io.IOException;
+import cs505.grad1.sensoragg.*;
+
+
+public class MockFactory implements AbstractAggregatedDataFactory {
```

Comment:
Your groups use of the Mock grove pi, factory, etc was actually a great use of showing the value of the abstract factory pattern and a very creative solution to the challenge of not always having (or wanting to use) the actual grove pi.  However, be sure that any code that makes it into a "production" stream has some comments to let other developers know what it is how it should be used (or why it shouldn't be used).